README.md

```diff
@@ -11,7 +11,18 @@ An example PR can be found [here](https://github.com/lewispb/rubymap/pull/1).
 
 ## Prerequisites
 
-`brew install geos`
+```bash
+brew install postgres
+brew install postgis
+brew install geos
+```
 
+Add PostGIS to postgres
```

Comment:
I think `rake db:gis:setup` should handle this :)

Reply:
TIL 👍